internal/cmd/gocdk/apply.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
Do you think that at some point we're going to have multiple packages here, or will a single package main contain the whole functionality of gocdk?

Reply:
I am uncertain. The launcher logic may end up being its own package at some point, but we figured it was better to start with a single package for now.